stadtbezirksbudget-frontend/tests/types/AntragListSort.test.ts

```diff
@@ -0,0 +1,139 @@
+import { describe, expect, test } from "vitest";
+import { DataTableSortItem } from "vuetify/framework";
+
+import {
+  antragListSortOptionFromSortItems,
+  antragListSortToSortDto,
+  antragListSortToSortItem,
+  createEmptyListSort,
+  sortOptionsByField,
+} from "../../src/types/AntragListSort";
+import { sortOptionsRecord } from "../../src/types/AntragListSortDefinitions";
+
+describe("AntragListSort", () => {
+  test("create empty AntragListSort returns empty AntragListSort", () => {
+    const emptySort = createEmptyListSort();
+    expect(Object.values(emptySort).every((v) => v === undefined)).toBe(true);
```

Comment:
```suggestion
    Object.values(emptySort).forEach((v) => {
      expect(v).toBeUndefined();
    });
```
I'd suggest this assertion to provide better error messages